test/plugin/arraySupport.test.js

```diff
@@ -33,12 +33,26 @@ const testArrs = [
   [2010, 6, 10]
 ]
 
-describe('parse array local', () => {
-  testArrs.forEach((testArr) => {
-    it(testArr, () => {
-      expect(dayjs(testArr).format())
-        .toBe(moment(testArr).format())
```

Comment:
we should keep the same return with moment.js. otherwise, it will be a huge breaking change.

Reply:
The fact is the same bug is present in « moment ». JavaScript Date parses month number from zero to eleven. But most of the time if not all time, we use month from 1 to 12. Excepting the same result as moment will be impossible unless moment fix it. Another alternative is to update the docs and let users know that they need to maybe change their habits and consider zero as first month and 11 as the last month in order to properly use arraySupport plugin. What would you suggest?